net/Ssl.hpp

```diff
@@ -68,9 +50,61 @@ class SslContext
     static void dynlockDestroy(struct CRYPTO_dynlock_value* lock, const char* file, int line);
 
 private:
-    static std::unique_ptr<SslContext> Instance;
-
     SSL_CTX* _ctx;
 };
 
+namespace ssl
+{
+class Manager
+{
+public:
+    static void initializeServerContext(const std::string& certFilePath,
```

Comment:
Would have done this with an enum, and an array rather than two sets of methods myself but ... ;-) fine for now.